src/lib/logical_query_plan/stored_table_node.hpp

```diff
@@ -11,14 +11,29 @@ namespace opossum {
 
 class LQPColumnExpression;
 
+/**
+ * Represents a Table from the StorageManager in an LQP
+ *
+ * Holds Column and Chunk pruning information.
+ */
 class StoredTableNode : public EnableMakeForLQPNode<StoredTableNode>, public AbstractLQPNode {
  public:
   explicit StoredTableNode(const std::string& table_name);
 
   LQPColumnReference get_column(const std::string& name) const;
 
-  void set_excluded_chunk_ids(const std::vector<ChunkID>& chunks);
-  const std::vector<ChunkID>& excluded_chunk_ids() const;
+  /**
+   * @defgroup ColumnIDs and ChunkIDs to be pruned from the stored Table.
+   * Both vectors need to be sorted and unique when passed to `set_pruned_{chunk/column}_ids()`
```

Comment:
Minor, fine if you don‘t want to change it: When reading „unique vector“, I had to make a second pass. Maybe „... and not contain duplicates“ is clearer.